### ISSUE TITLE
password when active user is not an admin

### ISSUE THREAD

**Issue by strider72** [lanayotech/vagrant-manager#108]
My everyday user account on OS X is not an admin, as a matter of general security. Some Vagrant machines require me to put in my Admin password – such as before altering the Hosts file on Up or Halt – but there's no opportunity to use a different user, so the password fails because I'm not an admin.

When doing this in Terminal, I would probably just su in before running vagrant up. Perhaps Vagrant Manager could allow something similar?

**Comment by abdusfauzi** [comment]
Agree. Is there anyway, I can give permission to Vagrant Manager? It would be very handy :+1:
